src/isar/state_machine/states/monitor.py

```diff
@@ -13,13 +13,13 @@
 )
 from robot_interface.models.exceptions.robot_exceptions import (
     ErrorMessage,
+    RobotCommunicationException,
     RobotCommunicationTimeoutException,
     RobotException,
     RobotRetrieveInspectionException,
     RobotTaskStatusException,
-    RobotCommunicationException,
 )
-from robot_interface.models.inspection.inspection import Inspection
+from robot_interface.models.inspection.inspection import InspectionResult
```

Comment:
Rename back to inspection, but now inspections are not created before the result is ready for uploading